spec/jekyll-feed_spec.rb

```diff
@@ -136,6 +138,31 @@
       expect(post.summary).to be_nil
     end
 
+    context "with site.lang set" do
+      lang = "en_US"
+      let(:overrides) { {"lang" => lang} }
+      it "outputs a valid feed" do
+        expect(feed.feed_type).to eql("atom")
+        expect(feed.feed_version).to eql("1.0")
+        expect(feed.encoding).to eql("UTF-8")
+        expect(feed.valid?).to eql(true)
+      end
+
+      it "outputs the correct language" do
+        expect(feed.lang).to eql(lang)
+      end
+
+      it "sets the language of entries" do
+        post = feed.items.first
+        expect(post.lang).to eql(lang)
+      end
+
+      it "renders the feed meta" do
```

Comment:
This is testing a link in the feed to the site; not the feed meta tag. Perhaps it could use a better name.